test/integration/use_int_test.go

```diff
@@ -195,6 +197,8 @@ func (suite *UseIntegrationTestSuite) TestShow() {
 		suite.Require().NoError(err)
 		cp.ExpectLongString(longPath)
 	}
+	cp.ExpectLongString(ts.Dirs.Cache)
+	cp.Expect("/exec")
```

Comment:
Is this going to work on Windows?

Reply:
Will run the test on Windows. IIRC we run them in bash on Windows so in theory it should work.